### ISSUE TITLE
Embeds - Web images & video

### ISSUE THREAD

**Issue by jinyuanqing** [visual-space/visual-editor#81]
# Embeds - Default support for web images and video #81
# VE Embeds - Move custom embed insertion code from demo page to the controller
# VE Embeds - Resize image, see if we can reuse mobile options for web.

---
web and windows cannot show image.
web tips:!kIsWeb "Please provide EmbedBuilder for Web"
i can not find way

**Comment by adrian-moisa** [comment]
Yes indeed, it's a known issue. One of the reasons I forked from Quill was the lack of default support for web images and videos. I don't have a clear explanation of why this is the maintainers of Quill did not add support. At some point I asked around but did not get any answer. 

At the moment I'm busy fixing other more urgent issues in the code base. But support for images on the web is top prio. So expect this to be fixed within a month or so. I'll update here once ready. It's funny that I documented all these issues but forgot to list the images issue. It was such an obvious issue that I did not even bother to document. Thanks for opening the ticket.

Temporary notes:
Embeds - Restore inline images
{
  "type": "image",
  "imageUrl": "assets/posts/space-brains/international-standard-payload-rack-lockers.jpg",
  "caption": "International Standard Payload Rack Lockers"
},

**Comment by jinyuanqing** [comment]
Thank you for your reply。

**Comment by adrian-moisa** [comment]
We've been busy refactoring the existing code base and adding support for highlights and markers. In the meantime we got enough experience to figure out what to do for the web. We will fix this one soon. Apparently images already work just fine on the web. We just need to enable them for web (there's a piece of code that blocks images on the web). As for video, we have to add a web player.

![unknown](https://user-images.githubusercontent.com/11160141/198101828-3ca8828f-a110-495f-b10b-5d49cc97103d.png)

Closed #21 Since it was a duplicate

References:

https://github.com/singerdmx/flutter-quill/issues/887
